spec/spec_helper_acceptance.rb

```diff
@@ -11,6 +11,12 @@
     end
     # refresh check if cache needs refresh on next yum command
     on host, 'yum clean expire-cache'
+
+    major = fact_on(host, 'os.release.major')
+
+    if major == '8'
+      on host, puppet('resource', 'package', 'glibc-langpack-en', 'ensure=installed')
+    end
```

Comment:
I don't think this is needed for DHCP to function. If it is, it should go into `spec/setup_acceptance_node.pp`
```suggestion
```

Reply:
It's not strictly necessary but does suppress some very annoying and repetitive warnings that clutter the test logs. I've moved it to `spec/setup_acceptance_node.pp` and on the next round of changes I push I will include a comment explaining why as well.